xrdsst/controllers/client.py

```diff
@@ -353,6 +353,12 @@ def find_client(self, clients_api, client_conf):
         return found_clients[0]
 
     @staticmethod
+    def find_all_clients(clients_api, show_members=False, internal_search=False):
+        try:
+            return clients_api.find_clients(show_members=show_members, internal_search=internal_search)
+        except ApiException as find_err:
+            BaseController.log_api_error('ClientsApi->find_clients', find_err)
+    @staticmethod
```

Comment:
This class also seems to have a couple of PEP8 warnings, for example this method does not have a blank line before the other method